docker-compose.yml

```diff
@@ -7,6 +7,21 @@ secrets:
 services:
   mailer:
     build: .
-    image: 'cyhy-mailer:latest'
+    image: 'dhsncats/cyhy-mailer:1.0.6'
     secrets:
       - source: database_creds
+        target: database_creds.yml
+    volumes:
+      - type: bind
+        source: /var/cyhy/weekly_reporting/report_archive/latest
+        target: /cyhy_reports
+      - type: bind
+        source: /var/cyhy/orchestrator/output/archive/latest/reporting/trustymail_reports
+        target: /trustymail_reports
+      - type: bind
+        source: /var/cyhy/orchestrator/output/archive/latest/reporting/pshtt_reports
+        target: /pshtt_reports
+      - type: bind
+        source: /var/cyhy/weekly_reporting/scorecards/latest
+        target: /cybex_report
```

Comment:
"cybex_report" should probably be "cybex_scorecard", in order to reinforce the semantic difference between reports (focused on a single stakeholder) and scorecards (summarize data across multiple stakeholders).

Reply:
OK, I can change that.